### ISSUE TITLE
[Pudge] Immortal "Scavenging Guttleslug" + "Rot". The ability's effect is partially broken.

### ISSUE THREAD

**Issue by Astiberon** [ValveSoftware/Dota2-Gameplay#22407]
### Cosmetic Item Name

Scavenging Guttleslug / Golden Scavenging Guttleslug

### Description

The ability's effect is partially broken.
There are no larvae.
Now:
![image](https://github.com/user-attachments/assets/b95f9d3e-828b-42e3-b90d-42a4a9e41a9c)
Then:
![image](https://github.com/user-attachments/assets/e7f0a053-6903-41ad-983c-4c2e37318de5)

### Example Match ID

-

**Comment by Ximplixity** [comment]
Just noticed this issue as well while trying to figure out what was overriding the effect, turns out it was a bug. Based on the comment I received from my Reddit post, it is associated with the Cull when crossing plane Particle Stem Operator which is supposed to kill particles that are no longer visible, instead, it culls them immediately.

**Comment by Astiberon** [comment]
> This issue is stale because it has been open for 365 days with no activity.

UP

**Comment by Weezy2014** [comment]
anyone found an answer to this issue

**Comment by Astiberon** [comment]
<img width="849" height="63" alt="Image" src="https://github.com/user-attachments/assets/ceebe53d-ec9c-4081-bc03-937b59e38d05" />

The bug is still not fixed

**Comment by Bu3nyy** [comment]
Yep, still broken. The slugs are completely missing on, on both the default and the golden version.

**Comment by HanX0823** [comment]
as of 2026-08-31, still not fixed
